Remotion/Web/Development.WebTesting/WebDriver/Factories/BrowserFactoryBase.cs

```diff
@@ -0,0 +1,68 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using Coypu;
+using Coypu.Drivers;
+using JetBrains.Annotations;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Remote;
+using Remotion.Utilities;
+using Remotion.Web.Development.WebTesting.BrowserSession;
+using Remotion.Web.Development.WebTesting.Configuration;
+
+namespace Remotion.Web.Development.WebTesting.WebDriver.Factories
+{
+  public abstract class BrowserFactoryBase : IBrowserFactory
+  {
+    public IBrowserSession CreateBrowser (ITestInfrastructureConfiguration testInfrastructureConfiguration)
```

Comment:
If i remember correctly, this will be a merge conflict when rebasing onto the trunk with the CommandTimeout changes. Just FYI if you didnt have it on the radar